singer-connectors/tap-mysql/requirements.txt

```diff
@@ -1 +1 @@
-pipelinewise-tap-mysql==1.4.3
+-e git+https://github.com/transferwise/pipelinewise-tap-mysql.git@AP-1116_log_based_with_gtid#egg=pipelinewise-tap-mysql
```

Comment:
I'll change this once this [PR](https://github.com/transferwise/pipelinewise-tap-mysql/pull/90) is merged.